src/Components/Header/index.tsx

```diff
@@ -1,16 +1,25 @@
 
-import { LeftArrow } from '../../assets/Icons'
-
-export type HeroProps = {
+export type HeaderProps = {
```

Comment:
why are you exporting types?

Reply:
removed the export this type will be local to the component